profiles/linkedin.py

```diff
@@ -11,43 +11,60 @@ def __init__(self):
 
     # Checks database for matches, and scrapes more if not enough
     def get_profiles(self, company, title, num_profiles):
+        """
+        Get profiles based on the company, title.
+        This calls the get_profiles in scraper.py
+        :param company: Company name we're searching.
+        :param role: Role we're searching for.
+        :param num_profiles: Required number of profiles to fetch and store.
+        :return: List of Query objects of the profiles stored in the Profiles database that corresponds to the company and title.
+        """
         # first check in the database to see there are enough num_profiles
         # then call get_linkedin_urls_google_search, and then pass those urls to the scraper
         # and then store in the db
         profiles = []
         exclude_urls = []
```

Comment:
No need for this declaration, and set below to `profiles = [...]`

Reply:
done